Runtime/Game/Requests/ClassRequests.cs

```diff
@@ -55,93 +61,107 @@ public class LootLockerLoadouts
         public LootLockerCommonAsset asset { get; set; }
     }
 
-    public class LootLockerUpdateCharacterRequest
+    public class LootLockerCreateClassRequest
     {
         public bool is_default { get; set; }
         public string name { get; set; }
-        public string type { get; set; }
+        public string character_type_id { get; set; }
     }
 
-    public class LootLockerCreateCharacterRequest
+    public class LootLockerUpdateClassRequest
     {
         public bool is_default { get; set; }
         public string name { get; set; }
-        public string character_type_id { get; set; }
+        public string type { get; set; }
```

Comment:
Our docs don't actually mention that you can set the type, nor what the parameter name for doing it would be. Are you sure about this change?

Reply:
This has been removed in the new pr!